src/search/mod.rs

```diff
@@ -10,24 +13,51 @@ pub mod indexing;
 
 #[derive(Error, Debug)]
 pub enum SearchError {
-    #[error("Error while connection to the MeiliSearch database")]
-    IndexDBError(),
-    #[error("Error while connecting to the local server")]
-    LocalDatabaseError(#[from] mongodb::error::Error),
-    #[error("Error while accessing the data from remote")]
-    RemoteWebsiteError(#[from] reqwest::Error),
-    #[error("Error while serializing or deserializing JSON")]
+    #[error("Error while connecting to the MeiliSearch database")]
+    IndexDBError(meilisearch_sdk::errors::Error),
```

Comment:
Adding `#[from]` is recommended to prevent inconsistency

Reply:
Neither of the two that don't have `#[from]` can/should have it.  Meilisearch's `Error` struct doesn't implement `std::error::Error`, so it can't have a `#[from]` impl, and the `CurseforgeImportError` is more specific and doesn't apply to all `reqwest` errors, so it shouldn't have it.